packages/itwin/measure-tools/tsconfig.json

```diff
@@ -1,6 +1,13 @@
 {
   "extends": "./node_modules/@itwin/build-tools/tsconfig-base.json",
+  "compilerOptions": {
+    "jsx": "react",
+    "module": "NodeNext",
+    "moduleResolution": "nodenext",
+    "target": "ES2020",
+    "outDir": "./lib"
```

Comment:
i wonder if we should leave it as `/lib/esm`

Reply:
That would only affect deep imports, which shouldn't be a thing in the first place.